lib/sidekiq/web/csrf_protection.rb

```diff
@@ -81,6 +81,11 @@ def session(env)
 
             https://guides.rubyonrails.org/api_app.html#using-session-middlewares
 
+            # example config sidekiq specific session
+            # https://github.com/mperham/sidekiq/wiki/Monitoring#rails-api-application-session-configuration
+            Sidekiq::Web.use ActionDispatch::Cookies
+            Sidekiq::Web.use ActionDispatch::Session::CookieStore, key: "_interslice_session"
```

Comment:
What is `_interslice_session`? Where does this code go?

Reply:
It's from link above https://guides.rubyonrails.org/api_app.html#using-session-middlewares and also i found this answer https://github.com/mperham/sidekiq/issues/4850#issuecomment-810880012